G_channels.md

```diff
@@ -231,6 +231,16 @@ end
 
 That's all there is to our basic chat app. Fire up multiple browser tabs and you should see your messages being pushed and broadcasted to all windows!
 
+#### Socket Assigns
+
+Similar to connection structs (`%Plug.Conn{}`) it is possible to assign values to a channel socket. `Phoenix.Socket.assign/3` is conveniently imported into a channel module as `assign/3`:
```

Comment:
I think instead of the parenthesis, we could have `%Plug.Conn{}` set off by commas.
. . . structs, `%Plug.Conn{}`, it is . . .